### PR TITLE
Hugs: new recipe

### PR DESCRIPTION
**Pull request by miqlas**
For tojoko/postmen.

### CODE REVIEW

**Comment by scottmc** [comment]
No need for a HUGS license, that is the BSD 4 Clause

File: dev-lang/hugs98/hugs98-dec2001.recipe [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+{
+	cd src/unix
+
+	libtoolize -vfi
+	runConfigure --omit-dirs 'docDir dataRootDir' ./configure \
+		--with-pthreads
+
+	cd ..
+	make $jobArgs OBJ_DIR=objects
+	make install
```
Comment (korli) [inline dev-lang/hugs98/hugs98-dec2001.recipe]: miqlas is this "make install" needed?

Comment (miqlas) [inline dev-lang/hugs98/hugs98-dec2001.recipe]: Nope...

**Comment by miqlas** [comment]
@scottmc, @korli : is it BSD 4 Clause or BSD 3 Clause then?
I see 3...

**Comment by korli** [comment]
It's 3, already fixed btw.

**Pull request merged**